docs/docs/custom-graph-components.mdx

```diff
@@ -487,3 +487,26 @@ which trains a new model:
 
 ```python (docs/sources/data/test_classes/custom_graph_components/nlu_sparse.py)
 ```
+
+### NLU Meta Learners
```

Comment:
I think this is good. My one feedback would be that we should start by clarifying that most of the time these components won't need to be trainable (e.g. if you have an intent classifier that sets the intent to `inform` if only an entity is present). 

So this is really just the most advanced / general case